group7_ws/src/auefinals/aue_finals/launch/turtlebot3_autonomy_final.launch

```diff
@@ -36,7 +36,7 @@
   <param name="robot_description" command="$(find xacro)/xacro --inorder $(find turtlebot3_description)/urdf/turtlebot3_$(arg model).urdf.xacro" />
 
   <node pkg="gazebo_ros" type="spawn_model" name="spawn_urdf" args="-urdf -model turtlebot3_$(arg model) -x $(arg x_pos) -y $(arg y_pos) -z $(arg z_pos) -Y $(arg yaw_pos) -param robot_description" />
-
+  <node pkg="aue_finals" type="wall_follower_obstacle_avoidance.py" name="control" output="screen"/>
```

Comment:
We talked about a new launch file with name = 'lidar_turtlebot3_autonomy_final.launch'. Please do not use this file to launch your task